src/main/java/com/cloudbees/groovy/cps/tool/Translator.java

```diff
@@ -125,7 +125,6 @@ public Translator(CompilationTask task) throws IOException {
         $CpsCallableInvocation = codeModel.ref("com.cloudbees.groovy.cps.impl.CpsCallableInvocation");
         $Builder               = codeModel.ref("com.cloudbees.groovy.cps.Builder");
         $CatchExpression       = codeModel.ref("com.cloudbees.groovy.cps.CatchExpression");
-        $DefaultGroovyMethods  = codeModel.ref("org.codehaus.groovy.runtime.DefaultGroovyMethods");
```

Comment:
We are actually processing multiple classes here so it was incorrect to hard-code this one.